materialrangebar/src/main/res/values/attrs.xml

```diff
@@ -32,6 +32,7 @@
         <attr name="mrb_pinRadius" format="dimension"/>
         <attr name="mrb_connectingLineColors" format="reference"/>
         <attr name="mrb_onlyOnDrag" format="boolean"/>
+        <attr name="mrb_pinsEnabled" format="boolean"/>
```

Comment:
we already have a property `mrb_temporaryPins` and i think this one should be changed so it should have the following three values
1. ALWAYS (This displays them all the time)
2. NEVER (This hides them  all the time)
3. ON_TOUCH_ONLY (This displays them when we are modifying the value and then hides them )


1 & 3 can be achieved right now by changing value for `mrb_temporaryPins` to `true` and `false` but the 2nd one is what you have to implement and at the same time you need to change the `mrb_temporaryPins`

So at the end the `mrb_temporaryPins` would be changed to something like this
```

 <attr name="mrb_temporaryPins" format="enum">
            <enum name="ALWAYS" value="0" />
            <enum name="NEVER" value="1" />
            <enum name="ON_TOUCH_ONLY" value="2" />
        </attr>
```

Reply:
makes sense, I will add it